### PR TITLE
Rename `build-this` to `build` and make it run the setup tasks

### PR DESCRIPTION
**Pull request by AlanGabbianelli**
See specific commit messages for more context
Paired with @thomasalanlee

### CODE REVIEW

File: lib/commands/build.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require_relative './base'
+require_relative './compose'
+
+class Commands::Build < Commands::Base
+  def call
+    check_service_exists
+    system.call("make", "-f", "#{config_directory}/Makefile", service)
```
Comment (benthorner) [inline lib/commands/build.rb]: Can we just do `make service` here?

Comment (thomasleese) [reply lib/commands/build.rb]: By passing the Makefile directly, you can run this from the app directory itself not just `govuk-docker`.

Comment (benthorner) [reply lib/commands/build.rb]: Oh, I thought that's what @barrucadu fixed in https://github.com/alphagov/govuk-docker/pull/54?

Comment (benthorner) [reply lib/commands/build.rb]: Ah, sorry - I see what you mean now. (It's not the service-specific one.)

**Pull request merged**